portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lindsey's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <div id="introduction">
+      <h1>Hey, I'm Lindsey.</h1>
+      <h2>It's nice to meet you.</h2>
+      <p>This is me.</p>
+      <img src="images/profilepic.JPG"/>
+    </div>
+
+    <div id="projects">
+    <p>Here are some of the projects I've been working on:</p>
+    <ul>
+      <li>CU@Home - an augmented reality snapchat filter that lets you
```

Comment:
Please prefer a newline for each list element. https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

Reply:
Got it.